src/test/resources/cucumber/features/componentTests/configure-sast-comment-script.feature

```diff
@@ -0,0 +1,29 @@
+@DeleteBranchFeature @ComponentTest
+Feature: Using Groovy script to configure SAST comment on scan request
+
+  Scenario Outline: Configuring SAST comment using a script result
+    Given given 'sast-comment' script name is '<comment script>'
+    When CxFlow Triggering sast scan
+    Then CxFlow scan comment is equal to '<comment>'
+
+
+    Examples:
```

Comment:
What about the case when a script has invalid syntax?

Reply:
Added